.github/workflows/controller.yaml

```diff
@@ -0,0 +1,90 @@
+name: MC
```

Comment:
Can we just call this `ci`?  It's small and doesn't take too much room in the actions UI, and it avoids `master`, which we got rid of throughout Spack a while back.

Reply:
Done.